mnswaleh/apisendIT
# #161859761 change delivery location

#### What does this PR do?
Fixes the PUT /parcel/<parcelId>/change-delivery endpoint to change delivery order instead of current location 

#### Description of Task to be completed?
should have the endpoint changin delivery location

#### How should this be manually tested?
1 on postman, enter url http://127.0.0.1:5000/api/v1/parcels/345616/change-delivery
2 send put request with desired delivery location
3 details of order 345616 changed
4 response code is 201
5 run pytest, the test should pass

#### What are the relevant pivotal tracker stories?
#161859761
#### Screenshots (if appropriate)
![screenshot from 2018-11-10 21-53-47](https://user-images.githubusercontent.com/39084014/48305179-11d60200-e537-11e8-899d-ed8af040954e.png)
